jwst/regtest/test_nircam_mtimage.py

```diff
@@ -11,7 +11,7 @@
 def test_nircam_image_moving_target_i2d(rtdata, fitsdiff_default_kwargs):
     """Test resampled i2d of moving target exposures for NIRCam imaging"""
     rtdata.get_asn("nircam/image/mt_asn.json")
-    rtdata.output = "mt_assoc_i2d.fits"
+    rtdata.output = "mt_asn_i2d.fits"
```

Comment:
What required this change?

Reply:
I reverted this because I think it's linked to the problems with the asn table and pool names.  Looking at the input association, it appears that the only place the string `"assoc"` appears anywhere is in `asn["products"][0]["name"]`.  I have to admit that the way our pipeline determines default filenames is completely opaque to me.  We'll see what the regression tests turn up, but it may be that library still needs to be modified in some way to update that field in a way that retains the old behavior with ModelContainer